pkg/backend/backend.go

```diff
@@ -64,6 +66,20 @@ type StackReference interface {
 	Name() tokens.QName
 }
 
+// PolicyPackReference is an opaque type that refers to a PolicyPack managedby a backend. The CLI
+// uses the ParsePolicyPackReference method to turn a string like "myOrg/mySecurityRules" into a
+// PolicyPackReference that can be used to interact with the PolicyPack via the backend.
+// PolicyPackReferences are specific to a given backend and different back ends may interpret the
+// string passed to ParsePolicyPackReference differently.
+type PolicyPackReference interface {
```

Comment:
Can we move this into the `httpstate` package?

Reply:
Yeah.

Reply:
I think we want to expose this through `PolicyPack#Ref()`. All of that could conceivably be a part of `httpstate` but I think it will complicate the code more than it's worth.